### ISSUE TITLE
Make "coregistered TBA DEM" available to tests and documentation.

### ISSUE THREAD

**Issue by erikmannerfelt** [GlacioHack/xdem#160]
Currently, whenever a test or documentation example needs a coregistered DEM, `NuthKaab` is run to produce it. This means that coregistration is run a handful of times instead of just once, every time the tests or documentation build is run.

Another approach would be a simple function in the examples module that either runs coregistration or loads an already existing dataset. The pseudocode would be:
```python
def get_coregistered_example():
    if not existing:
          dem = coregister_dem(...)  # Run the coregistration if it hasn't been made yet.
          dem.save("path/to/example.tif")  # Save it so that future calls won't need to recreate the file
          return dem
    else:
          return xdem.DEM("path/to/example.tif")  

```

**Comment by erikmannerfelt** [comment]
This was brought up in #159

**Comment by rhugonnet** [comment]
Closed by #161
